pkg/utils/utils.go

```diff
@@ -413,3 +413,25 @@ func GetAKONamespace() string {
 	}
 	return AKO_DEFAULT_NS
 }
+
+func GetTokenFromRestObj(robj interface{}, ctrlAuthtoken string) (string, bool) {
+	for _, aviToken := range robj.(map[string]interface{})["results"].([]interface{}) {
```

Comment:
Interface type should be checked before conversion.

Reply:
done